testing/mod.ts

```diff
@@ -102,6 +102,9 @@ export function test(t: TestDefinition | TestFunction): void {
 
 const RED_FAILED = red("FAILED");
 const GREEN_OK = green("OK");
+const RED_BG_FAIL = bgRed(" FAIL ");
+
+const VERBOSE = true;
```

Comment:
possible to consider adding an arg like the silent mode? example `--verbose` ?

Reply:
What's the purpose of this variable? It seems superfluous since it's always true

Reply:
@bartlomieju the purpose is to lhave an option in the code for devs to change it later on, if we know which approach to go for, then we dont need the variable.

Reply:
@zekth Yes! this is best alternative. ill add this for sure and get rid of the variable @bartlomieju